assets/javascripts/discourse/templates/components/user-profile-avatar.hbs

```diff
@@ -2,10 +2,11 @@
   {{animated-bound-avatar @user "huge"}}
   <UserAvatarFlair @user={{@user}} />
   {{! template-lint-disable no-capital-arguments }}
```

Comment:
```suggestion
```
(`@args` has now been replaced with `@outletArgs`, so this linting error is resolved 🎉)